frontend/src/lib/utils.tsx

```diff
@@ -519,6 +503,22 @@ export function humanFriendlyLargeNumber(d: number): string {
     return `${prefix}${d}`
 }
 
+/** Format currency from string with commas and a number of decimal places (defaults to 2). */
```

Comment:
out of curiosity, why move it?

Reply:
Ah, this is a leftover. I was moving other functions here but gave up, let me put this back to simplify diff

Reply:
Actually, rebasing is hard because of the generated files, I'll just keep it here for simplicity